### ISSUE TITLE
Training with GTA output from Tecotron I.

### ISSUE THREAD

**Issue by zhangjh915** [r9y9/wavenet_vocoder#96]
I am currently trying to use the predicted mel conditions from tacotron I to train the WaveNet but got strange results. The loss dropped dramatically from 6-ish to 1e-3 in a hundred steps and kept really small to approximately 1e-8 after that. The sounds generated were just noise.

I simply rescaled the predicted mel conditions to [0,1] and cut the end of the audio tensors to make sure their length equal to hop_size * len(mel). Other parameters like the frame shift were the same as Tacotron. 

I noticed that there is no trim_silence process in my Tacotron I code, and I wonder if it matters. Does the WaveNet have any requirement on the training data?

Any help or suggestion would be appreciated!

**Comment by duvtedudug** [comment]
My solution was to feed Wavenet preprocessed real specs in as ground truth. This way the Taco generated GTA mel specs are the right length for Wavenet.
